stdlib/http/client.pyi

```diff
@@ -83,6 +83,7 @@ class HTTPResponse(io.BufferedIOBase, BinaryIO):
     headers: HTTPMessage
     version: int
     debuglevel: int
+    fp: io.BufferedReader
```

Comment:
How did you come up with the `io.BufferedReader` type? Our stub for `sock.makefile` just has `BinaryIO`.

Reply:
I guess the stub must be wrong then.

the sock of which a `makefile()` is executed is either a `socket.socket` or an `ssl.SSLSocket` instance. Inspecting the result of a `makefile()` yields:
```python
>>> import socket
>>> a = socket.create_connection(('ksol.io', 80))
>>> a.__class__
<class 'socket.socket'>
>>> b = a.makefile('rb')
>>> b.__class__
<class '_io.BufferedReader'>
```